java/test/jmri/jmrit/display/MemoryIconTest.java

```diff
@@ -91,6 +91,7 @@ protected void init() {
         Assert.assertNotNull("Instance exists", i);
         jmri.InstanceManager.memoryManagerInstance().provideMemory("IM2").setValue("");
         to.setMemory("IM2");
+        to.setMemory("IM2");
```

Comment:
This is reverting something that was fixed on master a week or so ago.  This makes me suspect what's been included in these two commits...